### PR TITLE
CLDR-14925 Preliminary clean-up for snapshot of Priority Items Summary

### PR DESCRIPTION
**Pull request by btangmu**
-Call System.out.println, not System.err.println, unless it is an error

-Make some debugging output conditional on DEBUG or DEBUG_THREADS

-Move an URL to CLDRURLS.PRIORITY_SUMMARY_HELP_URL

-Call ph.shouldHide slightly sooner to avoid wasting time

-Move some code into a new subroutine recordEnglishChanged

-Fix deprecation warning for SurveyLog.logException

-Employ existing method getChoiceSetForOrg instead of duplicating code

-Remove dead code

-Make private where possible

-Formatting

-Comments

CLDR-14925

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
strange build error:
`   TestPaths {
    TestNonLdml
Error:  (TestPaths.java:437)  Error: Inconsistency:	file=annotations/dsb.xml	elementType=PCDATA	value=«»	path=//ldml/annotations/annotation[@cp="👨‍⚕"][@type="tts"]

 (4.023s) FAILED (1 failure(s))`

**Comment by srl295** [comment]
> strange build error: ` TestPaths { TestNonLdml Error: (TestPaths.java:437) Error: Inconsistency: file=annotations/dsb.xml elementType=PCDATA value=«» path=//ldml/annotations/annotation[@cp="👨‍⚕"][@type="tts"]
> 
> (4.023s) FAILED (1 failure(s))`

I saw similar in a PR but a different url.  Something must have gone in to make this unstable.

**Comment by srl295** [comment]
> > strange build error: ` TestPaths { TestNonLdml Error: (TestPaths.java:437) Error: Inconsistency: file=annotations/dsb.xml elementType=PCDATA value=«» path=//ldml/annotations/annotation[@cp="👨‍⚕"][@type="tts"]
> > (4.023s) FAILED (1 failure(s))`
> 
> I saw similar in a PR but a different url. Something must have gone in to make this unstable.

https://github.com/unicode-org/cldr/pull/1642/files#diff-62a59f099d43391ba60f3cdde57c4cbdd6f9adee4200e8730200b9707f4992bc look, similar error

**Comment by srl295** [comment]
> > > strange build error: ` TestPaths { TestNonLdml Error: (TestPaths.java:437) Error: Inconsistency: file=annotations/dsb.xml elementType=PCDATA value=«» path=//ldml/annotations/annotation[@cp="👨‍⚕"][@type="tts"] (4.023s) FAILED (1 failure(s))`
> > 
> > 
> > I saw similar in a PR but a different url. Something must have gone in to make this unstable.
> 
> https://github.com/unicode-org/cldr/pull/1642/files#diff-62a59f099d43391ba60f3cdde57c4cbdd6f9adee4200e8730200b9707f4992bc look, similar error



> > > strange build error: ` TestPaths { TestNonLdml Error: (TestPaths.java:437) Error: Inconsistency: file=annotations/dsb.xml elementType=PCDATA value=«» path=//ldml/annotations/annotation[@cp="👨‍⚕"][@type="tts"] (4.023s) FAILED (1 failure(s))`
> > 
> > 
> > I saw similar in a PR but a different url. Something must have gone in to make this unstable.
> 
> https://github.com/unicode-org/cldr/pull/1642/files#diff-62a59f099d43391ba60f3cdde57c4cbdd6f9adee4200e8730200b9707f4992bc look, similar error

Error doesn't repro for me locally on your PR (nor on mine). @macchiati do you recognize this err?

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java [hunk 1]
```diff
@@ -263,44 +264,36 @@ public boolean isStopped() {
...
+        private StringBuffer aBuffer = new StringBuffer();
 
-        public Task(QueueEntry entry, CLDRLocale locale, SurveyMain sm, String baseUrl, Level usersLevel,
+        private Task(QueueEntry entry, CLDRLocale locale, SurveyMain sm, String baseUrl, Level usersLevel,
             Organization usersOrg, final String st_org) {
             isSummary = isSummary(locale);
-            if (DEBUG)
-                System.err.println("Creating task " + locale.toString());
+            if (DEBUG) {
+                System.out.println("Creating task " + locale.toString());
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java:295]: better just `logger.debug(…)` (and drop `DEBUG`) but this is fine for now

**Review by srl295 — APPROVED** [review]
cleanup LGTM

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java:295]: i had a hunch you would suggest logger... however DEBUG is currently defined as DEBUG = false || CldrUtility.getProperty("TEST", false) ... not sure if that functionality is still desired and didn't want to experiment with it right now ...

**Comment by macchiati** [comment]
I ran into it as well, and filed
https://unicode-org.atlassian.net/browse/CLDR-15211

I don't know why it started showing up now, but a bunch of test failures
are hidden when we aren't in exhaustive mode.

It didn't repro on my local machine either.

Mark

**Comment by btangmu** [comment]
@macchiati @srl295 the TestNonLdml error persists when I re-run the checks for this PR.

Mark I see your new ticket https://unicode-org.atlassian.net/browse/CLDR-15211 -- if that's blocking progress, is it hard to fix or work around?

I don't understand very well about "exhaustive mode" -- how can I set the degree of exhaustiveness locally to match what it is on github? Or vice-versa?

**Comment by srl295** [comment]
You may need to rebase on main first. 

I didn't think that prs are checked in exhaustive mode, so the difference is mysterious. 

Exhaustive is just a setting, off by default

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/1646

**Pull request closed without merging**